site/docs/v1/tech/tutorials/integrate-react.md

```diff
@@ -0,0 +1,123 @@
+---
+layout: doc
+title: Integrate Your React Application With FusionAuth
+description: Integrate your React application with FusionAuth
+navcategory: getting-started
+prerequisites: nodejs
+technology: React
+language: JavaScript
+---
+
+## Integrate Your {{page.technology}} Application With FusionAuth
+
+{% include docs/integration/_intro.md %}
+
+## Prerequisites
+
+{% include docs/integration/_prerequisites.md %}
+
+## Download and Install FusionAuth
+
+{% include docs/integration/_install-fusionauth.md %}
+
+## Create a User and an API Key
+
+{% include docs/integration/_add-user.md %}
```

Comment:
This is not replacing `{language}`. In #2091, I've created a `site/_includes/docs/v1/tech/tutorials/_integrate-add-user.liquid` file to handle this. Maybe we should use it instead?

Also, we lost the cool `->` Unicode replacement now that we are using Markdown.

Reply:
Thanks. I changed this one to use your implementation. We'll probably have to figure out a way to merge the include file stuff, I think I was following your lead.

For the unicode stuff, I just spend 15 minutes trying to figure it out and didn't see where we actually do the replacement. This is all going to be moved over to a new static site generator, so I think we should tackle that then.